test/src/module/storage/azureTest.c

```diff
@@ -556,7 +556,7 @@ testRun(void)
                     "content-length: 0\n"
                     "date: <redacted>\n"
                     "host: %s\n"
-                    "x-ms-version: 2019-12-12\n"
+                    "x-ms-version: 2021-06-08\n"
```

Comment:
I assume this is some API version change that enables the features we're using for their storage protocol?

Reply:
Yes, this the minimum version that supports version listing with a delimiter, i.e. not having to list out every file in every subpath to get versions (yes, that's the way they implemented it originally).